from PyQt5 import QtCore, QtWidgets


class Ui_CalculatorMainWindow(object):
    def setupUi(self, CalculatorMainWindow):
        CalculatorMainWindow.setObjectName("CalculatorMainWindow")
        CalculatorMainWindow.setWindowModality(QtCore.Qt.NonModal)
        CalculatorMainWindow.resize(480, 330)
        CalculatorMainWindow.setMinimumSize(QtCore.QSize(480, 330))
        CalculatorMainWindow.setMaximumSize(QtCore.QSize(480, 330))
        CalculatorMainWindow.setStyleSheet("background-color: rgb(0, 39, 57);\n" "")
        CalculatorMainWindow.setTabShape(QtWidgets.QTabWidget.Rounded)
        self.central_widget = QtWidgets.QWidget(CalculatorMainWindow)
        self.central_widget.setObjectName("central_widget")
        self.lcd_number_main = QtWidgets.QLCDNumber(self.central_widget)
        self.lcd_number_main.setGeometry(QtCore.QRect(10, 10, 460, 80))
        self.lcd_number_main.setMinimumSize(QtCore.QSize(460, 80))
        self.lcd_number_main.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.lcd_number_main.setAutoFillBackground(False)
        self.lcd_number_main.setStyleSheet("background-color: rgb(0, 94, 0);")
        self.lcd_number_main.setFrameShape(QtWidgets.QFrame.WinPanel)
        self.lcd_number_main.setFrameShadow(QtWidgets.QFrame.Raised)
        self.lcd_number_main.setLineWidth(2)
        self.lcd_number_main.setMidLineWidth(0)
        self.lcd_number_main.setSmallDecimalPoint(True)
        self.lcd_number_main.setDigitCount(15)
        self.lcd_number_main.setSegmentStyle(QtWidgets.QLCDNumber.Flat)
        self.lcd_number_main.setProperty("intValue", 0)
        self.lcd_number_main.setObjectName("lcd_number_main")
        self.gridLayoutWidget = QtWidgets.QWidget(self.central_widget)
        self.gridLayoutWidget.setGeometry(QtCore.QRect(10, 100, 266, 221))
        self.gridLayoutWidget.setObjectName("gridLayoutWidget")
        self.grid_layout_nums = QtWidgets.QGridLayout(self.gridLayoutWidget)
        self.grid_layout_nums.setContentsMargins(0, 0, 0, 0)
        self.grid_layout_nums.setObjectName("grid_layout_nums")
        self.button_6 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_6.setObjectName("button_6")
        self.grid_layout_nums.addWidget(self.button_6, 1, 2, 1, 1)
        self.button_dot = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_dot.setObjectName("button_dot")
        self.grid_layout_nums.addWidget(self.button_dot, 3, 1, 1, 1)
        self.button_4 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_4.setObjectName("button_4")
        self.grid_layout_nums.addWidget(self.button_4, 1, 0, 1, 1)
        self.button_9 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_9.setObjectName("button_9")
        self.grid_layout_nums.addWidget(self.button_9, 0, 2, 1, 1)
        self.button_1 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_1.setObjectName("button_1")
        self.grid_layout_nums.addWidget(self.button_1, 2, 0, 1, 1)
        self.button_7 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_7.setObjectName("button_7")
        self.grid_layout_nums.addWidget(self.button_7, 0, 0, 1, 1)
        self.button_8 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_8.setObjectName("button_8")
        self.grid_layout_nums.addWidget(self.button_8, 0, 1, 1, 1)
        self.button_5 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_5.setObjectName("button_5")
        self.grid_layout_nums.addWidget(self.button_5, 1, 1, 1, 1)
        self.button_2 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_2.setObjectName("button_2")
        self.grid_layout_nums.addWidget(self.button_2, 2, 1, 1, 1)
        self.button_3 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_3.setObjectName("button_3")
        self.grid_layout_nums.addWidget(self.button_3, 2, 2, 1, 1)
        self.button_0 = QtWidgets.QPushButton(self.gridLayoutWidget)
        self.button_0.setObjectName("button_0")
        self.grid_layout_nums.addWidget(self.button_0, 3, 0, 1, 1)
        self.gridLayoutWidget_2 = QtWidgets.QWidget(self.central_widget)
        self.gridLayoutWidget_2.setGeometry(QtCore.QRect(290, 100, 176, 221))
        self.gridLayoutWidget_2.setObjectName("gridLayoutWidget_2")
        self.grid_layout_opers_main = QtWidgets.QGridLayout(self.gridLayoutWidget_2)
        self.grid_layout_opers_main.setContentsMargins(0, 0, 0, 0)
        self.grid_layout_opers_main.setObjectName("grid_layout_opers_main")
        self.button_del = QtWidgets.QPushButton(self.gridLayoutWidget_2)
        self.button_del.setObjectName("button_del")
        self.grid_layout_opers_main.addWidget(self.button_del, 0, 1, 1, 1)
        self.button_ac = QtWidgets.QPushButton(self.gridLayoutWidget_2)
        self.button_ac.setObjectName("button_ac")
        self.grid_layout_opers_main.addWidget(self.button_ac, 0, 2, 1, 1)
        self.button_result = QtWidgets.QPushButton(self.gridLayoutWidget_2)
        self.button_result.setObjectName("button_result")
        self.grid_layout_opers_main.addWidget(self.button_result, 3, 2, 1, 1)
        self.button_sum = QtWidgets.QPushButton(self.gridLayoutWidget_2)
        self.button_sum.setObjectName("button_sum")
        self.grid_layout_opers_main.addWidget(self.button_sum, 2, 1, 1, 1)
        self.button_div = QtWidgets.QPushButton(self.gridLayoutWidget_2)
        self.button_div.setObjectName("button_div")
        self.grid_layout_opers_main.addWidget(self.button_div, 1, 2, 1, 1)
        self.button_ans = QtWidgets.QPushButton(self.gridLayoutWidget_2)
        self.button_ans.setObjectName("button_ans")
        self.grid_layout_opers_main.addWidget(self.button_ans, 3, 1, 1, 1)
        self.button_mul = QtWidgets.QPushButton(self.gridLayoutWidget_2)
        self.button_mul.setObjectName("button_mul")
        self.grid_layout_opers_main.addWidget(self.button_mul, 1, 1, 1, 1)
        self.button_sub = QtWidgets.QPushButton(self.gridLayoutWidget_2)
        self.button_sub.setObjectName("button_sub")
        self.grid_layout_opers_main.addWidget(self.button_sub, 2, 2, 1, 1)
        CalculatorMainWindow.setCentralWidget(self.central_widget)

        self.retranslateUi(CalculatorMainWindow)
        QtCore.QMetaObject.connectSlotsByName(CalculatorMainWindow)

    def retranslateUi(self, CalculatorMainWindow):
        _translate = QtCore.QCoreApplication.translate
        CalculatorMainWindow.setWindowTitle(
            _translate("CalculatorMainWindow", "Калькулятор")
        )
        self.button_6.setText(_translate("CalculatorMainWindow", "6"))
        self.button_dot.setText(_translate("CalculatorMainWindow", "."))
        self.button_4.setText(_translate("CalculatorMainWindow", "4"))
        self.button_9.setText(_translate("CalculatorMainWindow", "9"))
        self.button_1.setText(_translate("CalculatorMainWindow", "1"))
        self.button_7.setText(_translate("CalculatorMainWindow", "7"))
        self.button_8.setText(_translate("CalculatorMainWindow", "8"))
        self.button_5.setText(_translate("CalculatorMainWindow", "5"))
        self.button_2.setText(_translate("CalculatorMainWindow", "2"))
        self.button_3.setText(_translate("CalculatorMainWindow", "3"))
        self.button_0.setText(_translate("CalculatorMainWindow", "0"))
        self.button_del.setText(_translate("CalculatorMainWindow", "DEL"))
        self.button_ac.setText(_translate("CalculatorMainWindow", "AC"))
        self.button_result.setText(_translate("CalculatorMainWindow", "="))
        self.button_sum.setText(_translate("CalculatorMainWindow", "+"))
        self.button_div.setText(_translate("CalculatorMainWindow", "/"))
        self.button_ans.setText(_translate("CalculatorMainWindow", "Ans"))
        self.button_mul.setText(_translate("CalculatorMainWindow", "*"))
        self.button_sub.setText(_translate("CalculatorMainWindow", "-"))
